### DISCUSSION TITLE
overleaf导入nju毕业论文模板报错

### DISCUSSION THREAD

**Discussion by YifuWang1119** [nju-lug/NJUThesis#257] [Q&A]
我在尝试将毕业论文模板的.zip文件导入overleaf之后报错，具体的报错信息为找不到.cls文件，但我的.cls文件是存在的，并且和.tex文件位于同一级目录（其实就是zip包自带的格式）

**Comment by atxy-blip** [comment]
我能保证最新的用户模板文件 [`njuthesis-user-v1.4.1.zip`](https://github.com/nju-lug/NJUThesis/releases/download/v1.4.1/njuthesis-user-v1.4.1.zip) 在上传后是可用的。但由于不清楚你具体的打包方式和文件结构，所以也没法直接给出解决办法。

可以补充报错和上传后项目文件列表的截图。

**Comment by YifuWang1119** [comment]
您好！

首先感谢您的回复和对模板的制作与维护。
我选择upload a project 之后直接将 njuthesis-user-v1.4.1.zip
<https://github.com/nju-lug/NJUThesis/releases/download/v1.4.1/njuthesis-user-v1.4.1.zip>
 拖到上传区，之后报错如附件。

祝好，
逸夫

**Reply by atxy-blip** [reply]
直接回复通知邮件会被匿名代发，在网页端是看不到附件的。我建议在[本网页下](https://github.com/nju-lug/NJUThesis/discussions/257#discussioncomment-9823593)进行回复的操作。

**Reply by YifuWang1119** [reply]
![Uploading 微信截图_20240626070956.png…]()

**Reply by atxy-blip** [reply]
你发的图裂了。如果你用 QQ 的话，或许可以来模板的 QQ 咨询群里讨论这个问题，群号是 991559926。
